components/migrate/MigrateIndex/MigrateContent.vue

```diff
@@ -157,34 +65,30 @@
 </template>
 
 <script setup lang="ts">
-import { NeoButton, NeoIcon, NeoTooltip } from '@kodadot1/brick'
+import { NeoIcon, NeoTooltip } from '@kodadot1/brick'
 import Identity from '@/components/identity/IdentityIndex.vue'
 import useMigrate from '@/components/migrate/migrate'
+import MigrateContentReady from '@/components/migrate/MigrateIndex/MigrateContentReady.vue'
+import MigrateContentWaiting from './MigrateContentWaiting.vue'
```

Comment:
`MigrateMigrateIndexMigrateContentWaiting`

Reply:
I think we use this naming if we want to auto-import the component. In this case, I define the component to `import MigrateContentWaiting` instead. Because the component is not shared for now

Auto-import component useful for shared component IMO, wdyt?

Reply:
> I think we use this naming if we want to auto-import the component. In this case, I define the component to `import MigrateContentWaiting` instead. Because the component is not shared for now

well If you change folder name/structure you can have better import 

ie: 
- `/migrate/index/Content.vue` => `MigrateIndexContent` will resolve to a better name for auto import


> Auto-import component useful for shared component IMO, wdyt?

yes but we can afford to use what is imported by default with Nuxt

Reply:
do what you think is best 💪

Reply:
updated here f751eac67395f88aa1e6056f34eecc1ddb104bcd